javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,135 @@
+const allNames = [];
+const name = "";
+
+const todoActivities = [];
+
+function getReply(command) {
+
+
+    const commandArr = command.split(' ');
+
+
+    if (allNames.includes(commandArr[4])) {
+        greeting = "Hello again " + commandArr[4];
+
+    }
+
+    else if ("Hello my name is" + commandArr[4] && commandArr.includes('Hello')) {
+        allNames.push(commandArr[4])
+
+        greeting = "Nice to meet you " + commandArr[4];
+
+    }
+    else if (command.includes('What is my name')) {
+        whatIsMyName = allNames.pop();
+        greeting = "You are " + whatIsMyName;
+    }
+    else if (command.includes('Add') && command.includes('to my todo')) {
+        let todos = commandArr.slice(1, -3)
+        todos = todos.join(' ');
+        greeting = todos + " added to your todo"
+        todoActivities.push(todos);
+    }
+
+    for (let i = 0; i < todoActivities.length; i++) {
+        if (command.includes('Remove') && command.includes(todoActivities[i])) {
+
+            greeting = "Removed " + todoActivities[i] + " from your todo"
+            todoActivities.splice(i, 1)
+        }
+    }
+    for (let j = 0; j < todoActivities.length; j++)
+        if (command.includes("What is on my todo?")) {
+            let remainingActivities = "";
+            remainingActivities = remainingActivities + todoActivities;
+            greeting = "You have " + todoActivities.length + ' todos ' + remainingActivities;
+
+        }
+
+    if (command.includes('What day is it today?')) {
+        let today = new Date();
+        let day = today.getDate();
+
+        let year = today.getFullYear();
+
+        let months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"]
+
+        let month = months[today.getMonth()];
+        greeting = day + " of " + month + " " + year;
+
+
+    }
+
+    if (command.includes('Calculate')) {
+
+        function toCalculate(num1, operator, num2) {
+
+            let result = 0;
+            if (operator === "+") {
+                result = num1 + num2;
+            }
+            if (operator === "-") {
+                result = num1 - num2;
+            }
+            if (operator === "*") {
+                result = num1 * num2;
+            }
+            if (operator === "/") {
+                result = num1 / num2;
+            }
+            greeting = result;
+        }
+        toCalculate(commandArr[1], '*', commandArr[3]);
```

Comment:
I think you want to do something like this
```suggestion
        toCalculate(commandArr[1], operator, commandArr[3]);
```

Reply:
Ohhh yes...it makes more sense